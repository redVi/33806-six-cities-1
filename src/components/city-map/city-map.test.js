import React from 'react';
import renderer from 'react-test-renderer';
import CityMap from '@/components/city-map/city-map.jsx';

describe(`CityMap`, () => {
  it(`renders correctly`, () => {
    const offers = [
      {name: `Canal View Prinsengracht`, price: 132, coordinates: [52.3909553943508, 4.85309666406198]},
      {name: `Nice, cozy, warm big bed apartment`, price: 180, coordinates: [52.369553943508, 4.85309666406198]},
    ];

    const tree = renderer.create(<CityMap offers={offers} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

