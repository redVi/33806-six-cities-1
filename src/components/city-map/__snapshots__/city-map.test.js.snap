// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityMap renders correctly 1`] = `
<section
  className="cities__map map"
  id="map"
/>
`;
